docs/source/install/migration.rst

```diff
@@ -31,13 +31,24 @@ RHEL / CentOS:
 
    sudo yum update $PKG_NAME
 
-4. Run Mistral database migration scripts:
+4. Upgrade Mistral database.
 
 .. sourcecode:: bash
 
+  # Stop related services
+  service mistral-api stop
+  service mistral stop
+  service postgresql stop
```

Comment:
Hm, I believe PostgreSQL should be running here, otherwise migration will fail, right (since migration populates db)?